### PR TITLE
Swodlr UI authentication

### PR DESCRIPTION
**Pull request by jbyrne6**
Changes:
- Earthdata Login implementation (not with middleware)
- Added to welcome page
- created utm advanced options toggle
- changed 'my data' page to display info in table
- added podaac navbar and footer
- added some initial contents to 'about' page

### CODE REVIEW

**Comment by frankinspace** [comment]
@jbyrne6 can you resolve the conflicts?

**Comment by jbyrne6** [comment]
> @jbyrne6 can you resolve the conflicts?

Yes will do. Github is not playing nice right now because it has been so long since I pulled changes form develop

**Comment by jbyrne6** [comment]
> > @jbyrne6 can you resolve the conflicts?
> 
> Yes will do. Github is not playing nice right now because it has been so long since I pulled changes form develop

@frankinspace, just pushed the fixes to the merge conflicts

File: src/user/authentication.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { TestAuthenticationResponse } from "../types/authenticationTypes";
+
+export const checkUserAuthentication = async () => {
+    const baseUri = 'https://d15gds5czd9p7k.cloudfront.net';
```
Comment (frankinspace) [inline src/user/authentication.ts:4]: probably want to pull this stuff out to a config

**Review by frankinspace — COMMENTED** [review]
I still can't get it to work locally but I need to dig a bit deeper still to find out the error (could be on my end).

**Comment by frankinspace** [comment]
I think each of these bullets could probably be their own PR. Still, great work!

**Comment by frankinspace** [comment]
Login issue was related to using Firefox: https://github.com/podaac/swodlr-ui/issues/27

File: src/user/authentication.ts (hunk 1 above)
Comment (frankinspace) [reply src/user/authentication.ts]: Will be addressed by https://github.com/podaac/swodlr-ui/issues/28

**Review by frankinspace — APPROVED** [review]

**Pull request merged**